src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/IFileBanner.cs

```diff
@@ -0,0 +1,215 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+#if CODE_STYLE
+using Microsoft.CodeAnalysis.Internal.Editing;
+#else
+using Microsoft.CodeAnalysis.Editing;
+#endif
+
+namespace Microsoft.CodeAnalysis.LanguageServices
+{
+    internal interface IFileBanner
```

Comment:
I'm not overly fussed, but `IFileBanner` sounds odd to me. How do you feel about `IFileBannerFacts` to continue the trend from whence this case?

Or just ignore the bikeshedding and merge :)

Reply:
i like that name.